### PR TITLE
add blog post about text packages

### PR DESCRIPTION
**Pull request by jeroen**
An update on the rOpenSci text processing suite.

### CODE REVIEW

File: _posts/2017-06-12-ropensci_text_tools.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+Also have a look at meta packages `readtext` or `textreadr` which import these and other tools for automatically reading text in many different formats. 
+
+## pdf utilities
+
+Our [pdftools](https://cran.r-project.org/web/packages/pdftools/index.html) package now supports reading pdf (extracting text or metadata) and rendering pdf to png, jpeg, tiff, or raw vectors on all platforms (incl. windows).
+
+```r
+# Read some text
+text <- pdftools::pdf_text('https://cran.r-project.org/doc/manuals/r-release/R-intro.pdf')
```
Comment (sckott) [inline _posts/2017-06-12-ropensci_text_tools.md]: this eg wouldn't run - s/text/pages

File: _posts/2017-06-12-ropensci_text_tools.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+categories: blog
+tags:
+- r
+- packages
+- software
+- opendata
+- pdf
+---
+
+At the rOpenSci [text workshop](http://textworkshop17.ropensci.org/) in April we discussed many ideas for improving text processing in R. Over the past weeks I was able to work on some of these suggestions and we released a few packages for reading and analyzing text.
```
Comment (stefaniebutland) [inline _posts/2017-06-12-ropensci_text_tools.md]: use this link for text workshop: https://ropensci.org/blog/blog/2017/05/03/textworkshop17

File: _posts/2017-06-12-ropensci_text_tools.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+- r
+- packages
+- software
+- opendata
+- pdf
+---
+
+At the rOpenSci [text workshop](http://textworkshop17.ropensci.org/) in April we discussed many ideas for improving text processing in R. Over the past weeks I was able to work on some of these suggestions and we released a few packages for reading and analyzing text. 
+
+Below an update on new and improved rOpenSci tools for working with text in R!
```
Comment (stefaniebutland) [inline _posts/2017-06-12-ropensci_text_tools.md]: Below **is** an update

File: _posts/2017-06-12-ropensci_text_tools.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+# Vectorized function
+text <- c("À chaque fou plaît sa marotte.", "猿も木から落ちる", 
+	"Алты́нного во́ра ве́шают, а полти́нного че́ствуют.", "Nou breekt mijn klomp!")
+
+cld2::detect_language(text)
+# [1] "fr" "ja" "ru" "nl"
+```
+
+Maëlle has written a [cool post](http://www.masalmon.eu/2017/06/10/rolandgarros/) comparing language classification methods using 18000 `"#RolandGarros2017"` tweets and Thomas [reminds us](http://notstatschat.tumblr.com/post/161449071226/stupid-word-games) that algorithms can easily be fooled. Still I found the accuracy on real text quite astonishing given the relatively small size of these libraries.
```
Comment (stefaniebutland) [inline _posts/2017-06-12-ropensci_text_tools.md]: use first and last names of people and link to their GitHub or twitter

File: _posts/2017-06-12-ropensci_text_tools.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+text <- antiword::antiword("https://jeroen.github.io/files/UDHR-english.doc")
+cat(text)
+### Lots of text...
+```
+
+Also have a look at meta packages `readtext` or `textreadr` which import these and other tools for automatically reading text in many different formats. 
+
+## pdf utilities
+
+Our [pdftools](https://cran.r-project.org/web/packages/pdftools/index.html) package now supports reading pdf (extracting text or metadata) and rendering pdf to png, jpeg, tiff, or raw vectors on all platforms (incl. windows).
```
Comment (stefaniebutland) [inline _posts/2017-06-12-ropensci_text_tools.md]: capitalize Windows

**Comment by sckott** [comment]
@jeroen nice! stef and I left some comments for you

Also, can you do more to set the context by stating the unmet need for these packages plus what already exists, and a bit more about what the motivation was for these updates?

**Comment by jeroen** [comment]
OK I have processed the comments and added an intro section with some words about text analysis in R.

**Pull request merged**